.github/workflows/check-spelling.yml

```
@@ -7,29 +7,33 @@
-            cspell_050923:filetypes/filetypes.txt
-            cspell_050923:fonts/fonts.txt
-            cspell_050923:fullstack/src/fullstack.txt
-            cspell_050923:rust/src/rust.txt
-            cspell_050923:typescript/src/typescript.txt
-          experimental_apply_changes_via_bot: 1
-          check_file_names: 1
+    spelling:
+        name: Spell checking
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)